pkg/lib/event/service.go

```diff
@@ -131,12 +136,19 @@ func (s *Service) makeContext(payload event.Payload) event.Context {
 		triggeredBy = event.TriggeredByTypeAdminAPI
 	}
 
+	ipAddress := httputil.GetIP(s.Request, bool(s.TrustProxy))
+	userAgent := s.Request.UserAgent()
+	clientID := clientid.GetClientID(s.Context)
```

Comment:
If the event is triggered by the portal, the IP address will be the cluster IP of the portal pod? Does it matter if we expose it in the event or we should hide it?

Reply:
Let me omit the IP address if the event is triggered by admin API.

Reply:
But on the other hand, we want to log the IP address calling the admin API if the caller is not ourselves. I guess we should not hide it at the moment. What do you think?